src/sdk/resources/resource.ts

```diff
@@ -12,10 +12,15 @@ export class Resource<TResource> {
 	protected resourcePath: string;
 	public defaultParams: any;
 
-	constructor(protected httpClient: IHttpClient, protected singularName: string, protected pluralName: string = undefined) {
+	constructor(protected httpClient: IHttpClient, protected singularName: string, protected pluralName: string = undefined, 
+				protected singularType?: string, protected pluralType: string = undefined) {
 		if (!this.pluralName) {
 			this.pluralName = `${this.singularName}s`;
 		}
+		
+		if (this.singularType && !this.pluralType) {
```

Comment:
Where is `pluralType` different than `pluralName`?

Reply:
In groups.ts, the pluralName is 'groups' while the pluralType is 'owners'.  Without this, I noticed that the groups were not read properly from the result